service-base/src/main/java/org/eclipse/hono/servicebase/HonoServiceBase.java

```diff
@@ -21,36 +21,47 @@
 
 import java.util.Objects;
 
-public abstract class HonoServerBase extends AbstractVerticle {
+public abstract class HonoServiceBase extends AbstractVerticle {
```

Comment:
OK, matches with the naming of AbstractVertxBasedHttpProtocolAdapter.
To be consistent makes much sense - will change it.